_topic_maps/_topic_map.yml

```diff
@@ -27,23 +27,24 @@ Topics:
 - Name: Updating OpenShift Service Mesh
   File: ossm-updating-openshift-service-mesh
 ---
-Name: Metrics
-Dir: metrics
+Name: Observability
+Dir: observability
 Distros: openshift-service-mesh
 Topics:
-- Name: Using metrics with Service Mesh
-  File: ossm-metrics-assembly
----
-Name: Distributed tracing
-Dir: traces
-Distros: openshift-service-mesh
-Topics:
-- Name: Configuring distributed tracing with Service Mesh
-  File: ossm-distr-tracing-assembly
----
-Name: Kiali Operator provided by Red Hat
-Dir: kiali
-Distros: openshift-service-mesh
-Topics:
-- Name: Using Kiali Operator provided by Red Hat
-  File: ossm-kiali-assembly
+- Name: Observability
+  File: ossm-observability-service-mesh-assembly
+- Name: Metrics and Service Mesh
+  Dir: metrics
+  Topics:
+  - Name: Using metrics with Service Mesh
+    File: ossm-metrics-assembly
+- Name: Distributed tracing and Service Mesh
+  Dir: traces
+  Topics:
+  - Name: Configuring distributed tracing with Service Mesh
+    File: ossm-distr-tracing-assembly
+- Name: Kiali Operator provided by Red Hat
+  Dir: kiali
+  Topics:
+  - Name: Using Kiali Operator provided by Red Hat
```

Comment:
The "provider by Red Hat" is needed on every occasion? It's the official name? If not, I think it could be dropped and the clause could be placed in the intro of an assembly.

Reply:
Yes it is needed as it is the official name, the way it is referenced in the UI, and the approved way from legal.